server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java

```diff
@@ -2180,6 +2180,7 @@ public void setObjectPermissions(
         if (CommonUtils.isEmpty(subjectIds) || CommonUtils.isEmpty(objectIds)) {
             return;
         }
+        Set<String> allSubjects = getAllSubjects();
```

Comment:
we can check whether this particular list of subjects exists, without reading the entire list